### PR TITLE
resolve moving screen

### PR DESCRIPTION
**Pull request by Madnessa**
We try to resolve delimitation screen but we don't change function of boundBy

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -52,7 +54,7 @@ body {
     text-align: center;
     padding-top: 30%;
     width: 100%;
-    height: 100%;
+    height: 100px;
```
Comment (Yoric) [inline css/style.css:57]: At the very least, please add a comment to explain this (CSS comments look like `/* ... */`)

**Comment by Yoric** [comment]
Almost good. Please add the fix mentioned in the patch.

**Comment by Yoric** [comment]
Also, it seems that you now need to rebase your patch.

**Comment by Yoric** [comment]
Any update?

**Comment by Madnessa** [comment]
We don't understand how rebase or pull our project in github, we have some mistake in terminal access.

**Comment by Yoric** [comment]
Well, then you need to fix these issues.

Most of the documentation you need is here: https://help.github.com/articles/fork-a-repo

If you need further help, you can either
- ask your professors;
- ask us on irc (server irc.mozilla.org, channel `#comete`, for instance, or `#frenchmoz`, or `#seneca`);
- come to our offices.

**Pull request closed without merging**